tools/genomespace/genomespace_importer.py

```diff
@@ -192,7 +191,7 @@ def download_from_genomespace_importer( username, token, json_parameter_file, ge
         # if using tmp file, move the file to the new file path dir to get scooped up later
         if using_temp_file:
             original_filename = filename
-            filename = ''.join( c in VALID_CHARS and c or '-' for c in filename )
+            filename = ''.join( c in FILENAME_VALID_CHARS and c or '-' for c in filename )
```

Comment:
My understanding is:

- For files going into Galaxy - it does seem that Dan explicitly wanted to allow spaces here and history items can include names with spaces (and most of our flagship tools generate items with spaces) so I don't know why would exclude them here. I didn't make that decision, but it seems reasonable.
- For downloads coming from Galaxy - I'm guessing we exclude spaces because they make the files easier to work with on the command-line. I didn't make that decision, but it seems reasonable.

So I don't think we need to be consistent about whitespace handling across these two different use cases. Can you explain more why you think they should be consistent - and if so do you want spaces in downloads or do you want to exclude spaces when importing from genomespace?